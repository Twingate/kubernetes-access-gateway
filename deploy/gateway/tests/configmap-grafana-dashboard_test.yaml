suite: Configmap Grafana Dashboard
templates:
  - configmap-grafana-dashboard.yaml
tests:
  - it: should create Grafana Dashboard
    set:
      metrics:
        grafanaDashboard:
          enabled: true
          labels:
            grafana_dashboard: "1"
    asserts:
      - matchSnapshot: {}
