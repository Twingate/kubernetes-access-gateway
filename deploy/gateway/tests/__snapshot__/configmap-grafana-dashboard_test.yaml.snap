should create Grafana Dashboard:
  1: |
    apiVersion: v1
    data:
      gateway-dashboard.json: |
        {
          "title": "Kubernetes Access Gateway Dashboard",
          "panels": [],
          "tags": [],
          "time": {
            "from": "now-1h",
            "to": "now"
          },
          "schemaVersion": 40,
          "version": 1
        }
    kind: ConfigMap
    metadata:
      labels:
        grafana_dashboard: "1"
      name: RELEASE-NAME-kubernetes-access-gateway-config-dashboard
